benchmark/inference/inference_benchmark.py

```diff
@@ -145,16 +174,39 @@ def run(args: argparse.ArgumentParser) -> None:
                                 time.reset()
                             with itt, profile:
                                 if args.full_batch:
-                                    full_batch_inference(model, data)
+                                    y = full_batch_inference(model, data)
+                                    if args.evaluate:
+                                        mask = data.test_mask
+                                        pred = y[mask].argmax(1)
+                                        test_acc = pred.eq(data.y[mask]).sum(
+                                        ).item() / mask.sum().item()
+                                        print(f'Full Batch Test Accuracy: \
+                                            {test_acc:.4f}')
                                 else:
-                                    model.inference(subgraph_loader, device,
-                                                    progress_bar=True)
+                                    y = model.inference(
+                                        subgraph_loader, device,
+                                        progress_bar=True)
+                                    if args.evaluate:
+                                        test_acc = model.test(
+                                            y, test_loader, device,
+                                            progress_bar=True)
+                                        print(f'Mini Batch Test Accuracy: \
+                                            {test_acc:.4f}')
```

Comment:
Maybe I am missing something, but why do we need to perform inference twice?

Reply:
`y` will be passed to `model.test`, so it will not perform inference twice. Spliting them is to record e2e time precisely.